group_vars/dataspace/staging.yml

```diff
@@ -11,23 +11,25 @@ java_type: jdk
 tomcat_user: "{{ deploy_user }}"
 tomcat_group: "{{ deploy_user }}"
 tomcat_install_path: "/home/{{ deploy_user }}"
-dspace_db_user: 'dataspace_staging_db_user'
+dspace_db_user: "dataspace_staging_db_user"
 dspace_db_password: "{{ vault_dataspace_staging_db_password }}"
 dspace_db_name: "dataspace_staging_db"
 dspace_db_port: "5432"
 ark_ezid_password: "{{ vault_ark_ezid_password }}"
 
+
 maven_version: 3.6.3
 ant_version: 1.10.8
 db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
 db_clusteradmin_user: "postgres"
 
 postgres_port: "{{ dspace_db_port }}"
-postgres_admin_user: "{{ db_clusteradmin_user }}"
-postgres_admin_password: "{{ db_clusteradmin_password }}"
+postgres_admin_user: "{{ dspace_db_user }}"
+postgres_admin_password: "{{ dspace_db_password }}"
 postgresql_is_local: false
-postgres_version: 12
-postgres_host: "gcp-postgres-staging1"
+postgres_version: 13
+postgresql_is_cloud: true
+postgres_host: "10.134.16.3"
```

Comment:
Should this IP address be migrated into a vault-encrypted file?

Reply:
I've never understood why we vault them at all beyond (we always did) this is also a non-routable IP that no one can get to who isn't allowed to.